biogeochem/EDPhysiologyMod.F90

```diff
@@ -2314,18 +2328,17 @@ subroutine CWDOut( litt, fragmentation_scaler, nlev_eff_decomp )
     integer :: c
     integer :: ilyr
     integer :: dcmpy
-    integer :: numlevsoil
     !----------------------------------------------------------------------
 
     do c = 1,ncwd  
 
        litt%ag_cwd_frag(c)   = litt%ag_cwd(c) * SF_val_max_decomp(c) * &
-             years_per_day * fragmentation_scaler
+             years_per_day * fragmentation_scaler(1)
```

Comment:
how did you choose index 1 for the frag_scalar?

Reply:
@ckoven and I reasoned that the above ground litters should use the scalars associated with the top of the soil layer.